### DISCUSSION TITLE
Disable login

### DISCUSSION THREAD

**Discussion by goose-ws** [jonaswinkler/paperless-ng#674] [Support]
Migrating from the original Paperless project to this wonderful fork, I was sad to learn that adding the Docker environment variable of `PAPERLESS_DISABLE_LOGIN=true` has no effect. I see it still referenced in the Docs, but I think that may be just from copying over the old docs.

For my situation, I have auth secured outside of paperless (No ports exposed, it's only reachable via an Nginx reverse proxy which requires auth).

In the interest of trying to streamline SSO/minimize duplicate logins so that my wife will actually use the stuff I set up, I'm hoping this feature can be re-implemented?

**Comment by jonaswinkler** [comment]
Wait, where is that referenced in the docs?

The implementation of that feature was *very* ugly (I'd call it a hack), and I've modified that.

The new, quite similar function is `PAPERLESS_AUTO_LOGIN_USERNAME=<username>`. You have to supply the name of a registered user, and that user will get logged in automatically.

**Reply by goose-ws** [reply]
Referenced in the old changelog here: https://paperless-ng.readthedocs.io/en/latest/changelog.html?highlight=PAPERLESS_DISABLE_LOGIN#id12

Thank you for the assistance!
